docs/GLOSSARY.md

```diff
@@ -1,76 +1,88 @@
 # Glossary
 
-Index of terminology used in Randovania. Certain terms sometimes are used only in the GUI, with more technical ones being used in code or CLI.
+Index of terminology used in Randovania. Some terms are used only in the GUI, with more technical ones being used in code or CLI.
 
 ## Area
 
-TODO
+A small chunk of a game's playable space, often only a single room. May contain one or more [Nodes](#node).
+
+Examples:
+- Phazon Mining Tunnel in Metroid Prime
+- Big Pink in Super Metroid
 
 ## Configuration
 
-TODO
+Has two meanings:
+
+- Colloquially, a term referring to the many types of settings a player may use when generating a game. This includes trick settings, item pool settings, starting items and location, [Gameplay and Cosmetic Patches](#game-patches), and many other things.
+
+- In code, the Configuration class that each game must provide, derived from the BaseConfiguration class. This is where game-specific parts of the [Preset](#preset) are defined.
 
 ## Dangerous Action
 
-TODO
+Any action that the [Generator](#generator) might take which could potentially leave it stranded; a one-way action. See also [Point of No Return](#point-of-no-return).
 
 ## Database
 
-TODO
+The collection of data that represents a game's [Logic](#logic), including the layout of [Worlds](#world) and [Areas](#area), the [Resources](#resource) a player has to progress, the locations of [Pickups](#pickup) and spawn points, and other information.
```

Comment:
> The source of truth of how a certain game is represented to Randovania. Contains a list of Worlds, a Resource Database, a Victory Condition and other smaller things.

Also add Resource Database:
> Defines all Resources relevant to this game. Can be visualized and modified in the Data Editor.

Reply:
That's a much more concise definition. I've incorporated parts of that.